zrml/swaps/src/root.rs

```diff
@@ -207,7 +207,7 @@ mod tests {
     fn calc_preimage_works_with_increasing_polynomial(value: u128, expected: u128) {
         // f(x) = 2x^3 - x^2 - x + 1 is positive and increasing on [1, \infty].
         let f = |x: u128| {
-            let third_order = bmul(_2, bpowi(x, 3)?)?;
+            let third_order = 2 * bpowi(x, 3)?;
```

Comment:
This looks like a pretty unnecessary change, but I guess I just felt like it made more sense to just multiply by 2 instead of going the roundabout way of using `2 * BASE` and then `bmul`.